packages/docs/src/examples/va-switch/CustomValue.vue

```diff
@@ -1,11 +1,13 @@
 <template>
+  {{ value }}
+  <br />
   <va-switch v-model="value" true-value="Agree" false-value="Disagree" />
 </template>
 <script>
 export default {
   data () {
     return {
-      value: true,
+      value: 'Agree',
```

Comment:
very minor thing, but something like `left` | `right` would make more sense as values.
Agree | Disagree look like translated strings to me, and that's not something you may want to pass to true-value or false-value.